server/controller/src/main/java/ai/starwhale/mlops/datastore/impl/MemoryTableImpl.java

```diff
@@ -145,7 +146,7 @@ private void load() {
     }
```

Comment:
Try loading the previous file when fail?

Reply:
done